Added description to show different ways of how to add human feedback (#17839)
cloudflare/cloudflare-docs: src/content/docs/ai-gateway/evaluations/add-human-feedback-api.mdx

---

This guide will walk you through the steps of adding human feedback to an AI Gateway request using the Cloudflare API. You will learn how to retrieve the relevant request logs, and submit feedback using the API.

> If you prefer to add human feedback via the dashboard,](/ai-gateway/evaluations/add-human-feedback/) ypu can learn how to on this tutorial.

Reviewer: ```suggestion

If you prefer to add human feedback via the dashboard, refer to [Add Human Feedback](/ai-gateway/evaluations/add-human-feedback/).
```
